src/Validate/PriceConstraint.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Drupal\tide_event\Validate;
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Form API callback. Validate element value.
+ */
+class PriceConstraint {
+
+  /**
+   * Validates given element.
+   *
+   * @param array $element
+   *   The form element to process.
+   * @param \Drupal\Core\Form\FormStateInterface $formState
+   *   The form state.
+   * @param array $form
+   *   The complete form structure.
+   */
+  public static function validate(array &$element, FormStateInterface $formState, array &$form) {
+    $error = FALSE;
+    $webformKey = $element['#webform_key'];
+    $price_from_value = $formState->getValue('price_from');
+    $price_to_value = $formState->getValue($webformKey);
+
+    error_log("Price From: " . $price_from_value, 0);
```

Comment:
@enzolutions why do you use `error_log` here?

Reply:
Sorry that was my frustration with CI , I will remove it